### PR TITLE
added-basic-information-about-javascript-and-links

### PR DESCRIPTION
**Pull request by rajat2502**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/index.md [hunk 1]
```diff
@@ -3,11 +3,12 @@ title: Basic JavaScript
 ---
 ## Basic JavaScript
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/quadratic-equations/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Javascript (often abbreviated as "JS") , invented by "Brendan Eich" , is a high level,dynamic programming language which when combined with HTML can convert static websites into Dynamic Websites(having some functionality).It is one of the most versitile language of all time , finding applications in almost all the domains from Machine Learning to Data Scarping and Web Development , but most of its uses are in the Web Development.
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/index.md]:
```suggestion
JavaScript (often abbreviated as "JS"), invented by "Brendan Eich", is a high level, dynamic programming language which when combined with HTML can convert Static Websites into Dynamic Websites (having some functionality). It is one of the most versatile languages of all time, finding applications in almost all domains from Machine Learning to Data Scraping and Web Development.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
I don't think we want to push campers away from the site for an overview in a Guide article for a challenge intro section?

**Comment by rajat2502** [comment]
I have made the required changes please do the review again.

**Review by jamiehaywood — APPROVED** [review]

**Comment by thecodingaviator** [comment]
The PR adds info about JS but nothing about what the curriculum would teach to the campers in general. How do you think about adding something @ojeytonwilliams @moT01 @RandellDawson ?

**Comment by ojeytonwilliams** [comment]
I agree with @thecodingaviator.  This is the guide article for the challenges, not JavaScript in general.  This article should probably be something like

> This section contains guides to help you better understand individual lessons from the [curriculum](https://learn.freecodecamp.org/).  To learn more about JavaScript in general, take a look at its section in the [guide](https://guide.freecodecamp.org/javascript)

The more general description in this PR could be incorporated into the main JavaScript article, but I don't think it belongs here.

**Comment by RandellDawson** [comment]
Hi @rajat2502 ,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**